config/values/recruitmentcycle.yml

```
@@ -0,0 +1,8 @@
+recruitmentcycle:
+  coursestart: September 2025
+  alltimesanddates: from 9am on 10 October 2024 to 6pm on 16 September 2025
+  openingdateonly: 10 October 2024
```

```suggestion
  openingdate: 8 October 2024
```

would maybe suggest shortening this?

also - more importantly! - isn't this supposed to be **8** October 2024?